### DISCUSSION TITLE
いいね済み＝緑、ストック済み＝グレーのまま、なのでわかりにくい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#409] [要望・アイデア・不具合]
https://github.com/increments/qiita-discussions/discussions/168 に関連する話です。
いいね/LGTMボタンは改善されましたが、ストックボタンに関してはリアクション前もリアクション後もグレーのままなので状態の区別が付きづらいです。

未いいね、未ストックの場合
<img width="96" alt="Screenshot 2023-01-01 at 17 12 26" src="https://user-images.githubusercontent.com/1148320/210164723-f54b3ac1-94de-44c0-bf3e-623fbc7663ec.png">

いいね済み、ストック済みの場合
<img width="115" alt="Screenshot 2023-01-01 at 17 12 46" src="https://user-images.githubusercontent.com/1148320/210164729-e254ffc8-37e6-43fa-a5ff-831e26144ab2.png">

後者に関しては「グレーだからまだストックしてなかったかな？」と勘違いしてもう一度ストックボタンをクリックしてしまいました。
（上の画像は2つ並んでいるので違いがわかりやすいですが、後者の画像単体で見たときはいいねだけが緑なので、ストックは「未ストック」のように見えます）

どちらのボタンも以下のように「リアクション前ならグレー」、「リアクション後なら緑」と統一した方が識別しやすいと思うのですがいかがでしょうか？

<img width="96" alt="Screenshot 2023-01-01 at 17 12 26" src="https://user-images.githubusercontent.com/1148320/210164723-f54b3ac1-94de-44c0-bf3e-623fbc7663ec.png">

<img width="110" alt="Screenshot 2023-01-01 at 17 25 30" src="https://user-images.githubusercontent.com/1148320/210165007-59ce56a3-af12-47b0-9257-468029b35aea.png">

**Comment by kskwtnk** [comment]
@JunichiIto 
ご意見ありがとうございます！
両方が緑なのは、それはそれでデザインルールなどとバッティングしてしまうこともあるのですが、見せ方そのものの変更は検討します！
